corehq/messaging/smsbackends/amazon_pinpoint/views.py

```diff
@@ -0,0 +1,35 @@
+import json
+from urllib.request import urlopen
+from corehq.apps.sms.api import incoming as incoming_sms
+from corehq.apps.sms.views import IncomingBackendView
+from corehq.messaging.smsbackends.amazon_pinpoint.models import SQLPinpointBackend
+from django.http import HttpResponse
+
+
+class PinpointIncomingMessageView(IncomingBackendView):
+    urlname = 'pinpoint_sms'
+
+    @property
+    def backend_class(self):
+        return SQLPinpointBackend
+
+    def post(self, request, api_key, *args, **kwargs):
+        request_body = json.loads(request.body)
+        message = request_body.get('Message')
+        subscribe_url = request_body.get('SubscribeURL')
+        if subscribe_url:
+            urlopen(subscribe_url)
+        elif message:
```

Comment:
Are `message` and `SubscribeURL` mutually exclusive? If not this should use `if` instead of `elif`.

Reply:
yes they are  mutually exclusive.